db/migrate/20230929084821_add_verification_permission_to_staff_users.rb

```diff
@@ -0,0 +1,9 @@
+class AddVerificationPermissionToStaffUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :staff,
+               :verification_permission,
```

Comment:
I'm wondering about the name here, the others are named as an action that can be taken if the user has the write permission (e.g. they can _withdrawn_ or they can _reverse decision_).

Perhaps `verify_permission` would better fit with the existing conventions?

Reply:
Should be addressed now =)